lib/apiservers/engine/backends/container.go

```diff
@@ -142,8 +150,8 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 	var err error
 
 	//TODO: validate the config parameters
-	log.Debugf("config.Config = %+v", config.Config)
-
+	log.Printf("Image fetch section - Container Create")
+	log.Printf("config.Config = %+v", config.Config)
```

Comment:
This looks useful.  But is it redundant?  I thought we print out all this stuff somewhere already when we create the container.  Could be wrong.